i18n/en.pot

```diff
@@ -140,7 +140,7 @@ msgstr ""
 msgid "Period"
 msgstr ""
 
-msgid "Organisation units"
+msgid "Organisation Units"
```

Comment:
Just curious, are these pot string case changes caused by a previous PR? I don't see the corresponding string changing in the js files.

Reply:
In src/components/edit/BoundaryDialog.js

<img width="1050" alt="Screenshot 2020-10-20 at 11 07 09" src="https://user-images.githubusercontent.com/548708/96565289-7a51c000-12c4-11eb-89b9-e17f0793b4d3.png">

Reply:
Fair enough, so the js change isn't part of this PR, no worries 👍   Classic case of "en.pot lag" then 🙂